foundations/javascript_basics/clean_code.md

```diff
@@ -100,7 +101,8 @@ This section contains helpful links to related content. It isn’t required, so
 * [Chaining methods to write sentences](https://web.archive.org/web/20190211152543/https://javascriptissexy.com/beautiful-javascript-easily-create-chainable-cascading-methods-for-expressiveness/)   
 
 ### Knowledge check
```

Comment:
Can you update this heading so that "Check" begins with a capital letter? Looks like the lowercase "c" might be causing an issue with rendering a jump link on the side.

Reply:
Hey, I had noticed you receiving help for this on the Discord. Just wanted to check whether you're good or if you needed any additional help 🙂

Reply:
Hey, 

I am not able to understand what needs to be done here, **_"Additionally, the "Closes" keyword should only link an issue to a PR, not another PR to a PR. That needs to be updated to link to the original issue."_**

Apart from this, I made the changes you had request in the clean code file (check --> Check)

Reply:
Ah gotcha. So what the `Closes` keyword does it links an issue with a PR, so that when a PR gets merged the issue automatically closes (rather than having to merge a PR then go to the issue to manually close it).

Right now, you have `Closes #23966`, which is linking this PR to your old PR. You just need to edit your original post in this PR so that it is `Closes #23935`.